be/src/storage/lake/schema_change.cpp

```diff
@@ -364,6 +368,16 @@ Status SchemaChangeHandler::convert_historical_rowsets(const SchemaChangeParams&
         }
     }
 
+    // copy delete vector files if necessary
+    if (op_schema_change->linked_segment() && base_metadata->has_delvec_meta()) {
+        for (const auto& delvec : base_metadata->delvec_meta().delvecs()) {
+            auto src = base_tablet->del_location(tablet_delvec_filename(base_tablet->id(), delvec.page().version()));
+            auto dst = new_tablet->del_location(tablet_delvec_filename(new_tablet->id(), delvec.page().version()));
+            RETURN_IF_ERROR(fs::copy_file(src, dst));
```

Comment:
if the uuid path is recorded in `DelvecPagePB` instead of version, copy is not needed?

Reply:
Good idea. Current delvec file's name is {tablet_id}_{version}.delvec, we can change it to {uuid}.delvec, and record it's name in `DelvecPagePB`. And then base_tablet and new_tablet can shared this delvec file.